boards/arm/atsamr21_xpro/doc/index.rst

```diff
@@ -151,6 +151,8 @@ externally connected SPI devices.
 | SLP_TR      | PA20 (OUT, GPIO output)                                                                  |
 +-------------+------------------------------------------------------------------------------------------+
 
+For example of this use check :ref:`wpanusb-sample` sample.
```

Comment:
An article and  comma would help here: ``For an example of this use, check ...``

Reply:
done!